scripts/ci/jsonSchemas/composer_block.json

```diff
@@ -47,6 +47,14 @@
     "type": {
       "type": "string",
       "description": "A type for this block: text, title, ulist, olist, etc"
+    },
+    "style": {
+      "type": "string",
+      "description": "A style required for button composer blocks"
+    },
+    "javascript": {
+      "type": "string",
+      "description": "A javascript file that performs an action for button compsoer blocks"
```

Comment:
@bakulf is there anything I should do here to specify that this is specifically for button composer blocks? Should I create new schema for buttons?